<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileSearchDialog</class>
 <widget class="QDialog" name="FileSearchDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>571</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../res/resources.qrc">
    <normaloff>:/default_icons/edit.png</normaloff>:/default_icons/edit.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>6</number>
      </property>
      <item row="2" column="2" colspan="2">
       <widget class="QLineEdit" name="keywordLineEdit">
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="directoryLineEdit">
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>9</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Keyword</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Folder</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Setting name</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QToolButton" name="openToolButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QLineEdit" name="nameLineEdit"/>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>9</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="searchAdditionalConditions">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QWidget" name="searchAdditionalConditionsPage">
      <attribute name="title">
       <string>Additional Conditions</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="horizontalSpacing">
          <number>0</number>
         </property>
         <item row="0" column="1" colspan="3">
          <widget class="QLineEdit" name="regexLineEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="6" column="3">
          <widget class="QComboBox" name="createComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>after</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>equals</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>before</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QCheckBox" name="fsizeCheckBox">
           <property name="text">
            <string>Filesize</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1" colspan="2">
          <widget class="QDateTimeEdit" name="modifiedDateTimeEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="displayFormat">
            <string>yyyy/MM/dd HH:mm:ss</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="recursiveCheckBox">
           <property name="text">
            <string>Folder recursive search</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QCheckBox" name="modifiedCheckBox">
           <property name="text">
            <string>Modified</string>
           </property>
          </widget>
         </item>
         <item row="9" column="2">
          <widget class="QComboBox" name="bytesComboBox1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>Bytes</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>KB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>MB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>GB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>TB</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QCheckBox" name="createCheckBox">
           <property name="text">
            <string>Created</string>
           </property>
          </widget>
         </item>
         <item row="10" column="3">
          <widget class="QComboBox" name="fsizeComboBox2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="currentText">
            <string> -</string>
           </property>
           <property name="currentIndex">
            <number>5</number>
           </property>
           <item>
            <property name="text">
             <string>over</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>or more</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>equals to</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>or less</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>under</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string> -</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>9</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="secondsCheckBox">
           <property name="text">
            <string>From search time</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QComboBox" name="secondsComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>s ago</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>min ago</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>h ago</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QLineEdit" name="fsizeLineEdit1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="7" column="3">
          <widget class="QComboBox" name="modifiedComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>after</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>equals</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>before</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QLineEdit" name="fsizeLineEdit2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="4" column="1" colspan="2">
          <widget class="QLineEdit" name="secondsLineEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>9</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="6" column="1" colspan="2">
          <widget class="QDateTimeEdit" name="createDateTimeEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="displayFormat">
            <string>yyyy/MM/dd HH:mm:ss</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="10" column="2">
          <widget class="QComboBox" name="bytesComboBox2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>Bytes</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>KB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>MB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>GB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>TB</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="regexCheckBox">
           <property name="text">
            <string>Regular Expression</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>9</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="9" column="3">
          <widget class="QComboBox" name="fsizeComboBox1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>over</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>or more</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>equals to</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>or less</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>under</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="8" column="0">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>9</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="testSearch">
     <widget class="QWidget" name="testSearchPage">
      <attribute name="title">
       <string>Search Test</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="2">
          <widget class="QPushButton" name="testSearchButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="SearchDialogTable" name="resultTableWidget">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLineEdit" name="directoryTestLineEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SearchDialogTable</class>
   <extends>QTableWidget</extends>
   <header>searchdialogtable.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>keywordLineEdit</tabstop>
  <tabstop>directoryLineEdit</tabstop>
  <tabstop>openToolButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../res/resources.qrc"/>
  <include location="../../res/resources.qrc"/>
  <include location="../../res/resources.qrc"/>
  <include location="../../res/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FileSearchDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>540</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FileSearchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>540</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
